src/style.css

```diff
@@ -0,0 +1,86 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 400;
+}
+
+html {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+}
+
+.search-form {
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  margin-top: 20px;
+  box-sizing: border-box;
```

Comment:
it's better to set for each element on the page (use `*` selector)
note it's the only property allowed to be set in `*` selector